src/init.cpp

```diff
@@ -349,9 +361,12 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-walletnotify=<cmd>", _("Execute command when a wallet transaction changes (%s in cmd is replaced by TxID)"));
     strUsage += HelpMessageOpt("-zapwallettxes=<mode>", _("Delete all wallet transactions and only recover those parts of the blockchain through -rescan on startup") +
         " " + _("(1 = keep tx meta data e.g. account owner and payment request information, 2 = drop tx meta data)"));
-                    
+
 #endif
 
+    strUsage += HelpMessageGroup(_("ZeroMQ publisher options"));
+    strUsage += HelpMessageOpt("-zmq<type><format><object>=<address>",_("Enable notification through zmq socket <address> (type=pub, format=hash|raw, object=block|transaction)"));
```

Comment:
It looks like that this is not implemented like that. Example: `zmqpubrawblock` is not supported. Only `zmqpubhashblock` and `zmqpubhashtransaction` is supported IMO. At least the should be a warning or error if you try to add a zmq channel which is not supported.